SafeguardSessions/CONTRIBUTION.md

```diff
@@ -2,43 +2,43 @@
+Both require a dedicated [Power Query SDK] extension to be able to create and build data connector projects. However, Microsoft suggests using Visual Studio Code, as the Visual Studio Code Power Query SDK will eventually become the default and maintained SDK.
 
-The following steps are required to set up a Visual Studio Code development environment:
+To set up a Visual Studio Code development environment:
 
 - Install [Visual Studio Code]
 - Install [Visual Studio Code Power Query SDK] (This will also install the Power Query / M Language extension.)
 
-To make development easier you can use `make` commands for building the connector and running the tests. In order to install `make` on Windows, do the following:
+To make development easier, you can use `make` commands for building the custom connector and running the tests. In order to install `make` on Windows:
 
 - Install [Chocolatey]
-- Install make via Chocolatey by running the following command: `choco install make`
+- Install `make` via Chocolatey by running the following command: `choco install make`
 
 ### Build the connector from source code
 
-1. Obtain the source from the GitHub repository
+1. Obtain the source code from the GitHub repository
 2. Install [MSBuild] for Windows
-3. Ensure MSBuild is included in your `PATH` variable.
-4. Build the source code with one of the options:
-    - In Visual Studio Code: Go to **Terminal > Run Build Task...** and select **Build connenctor project using MSBuild**
+3. Make sure that MSBuild is included in your `PATH` variable.
+4. Build the source code with one of the following options:
+    - In Visual Studio Code: Go to **Terminal > Run Build Task...**, and select **Build connenctor project using MSBuild**
     - From the command line: use the `make build` command
 
 ### Testing
 
-Running tests is only available from the command line as the test framework has been decoupled from the connector code. The tests and the unit test framework can be found under the test subfolder. This is a separate Power Query SDK project, so it is also necessary to be built. The `make build` command also builds the test project.
+The test framework has been decoupled from the connector code, so running tests is only available from the command line. You can find the tests and the unit test framework under the `Test` subfolder. This is a separate Power Query SDK project, so it requires building separately. The `make build` command also builds the test project.
 
-You can run the tests with `PQTest.exe` which comes with the Power Query SDK. By default, it is located in the SDK's extension folder, for example:
+You can run the tests with the `PQTest.exe`, which comes with the Power Query SDK. By default, it is located in the SDK's extension folder, for example:

```

Comment:
I don't agree with thes additional "the", but the TC/TW team suggested it, so it can stay.

Reply:
I asked for the reasoning behind the additional "the" . I'll come back with the answer. Maybe we can leave it out. :slightly_smiling_face:

Reply:
Okay.

I'll approve the PR but won't merge it yet. I'll leave this conversation unresolved.

Reply:
I have agreed with Anna that it can be left out, so I updated the sentence.

Conversation and reasoning:
> [8:40 AM] Anna Nemeth-Szabo (anemeths)
> 
> Szia!
> 
> ... 
> 
> A 'the' kérdés: 2 esetben tudom elképzelni, hogy ne kelljen ide a határozott névelő: 
> 
>  
> 
> 1.Ha voltaképp "általánosságban véve" beszélünk erről az exe-ről, és nem tudjuk konkrétan, melyikről van szó (itt a szövegkörnyezetből, akár a fenti a, akár a fenti b, eset van, úgy veszem ki, hogy tudjuk, melyik kell nekünk - de could be wrong)
> 
>  
> 
> 2.Ha inkább olyasformán emlegetjük, mint egy nagyon közismert appot, most hasraütve pl. mint a Microsoft Teams. Tulajdonnévként végülis LEHET értelmezni az .exe nevét (PQTest.exe), de akkor ahhoz szerintem nagyon kell tudni, hogy ez mennyire széles körben használt. 
> 
> ...
> 
> 
> [8:55 AM] David Dusa (ddusa)
> 
> Jó reggelt!
> 
> ...
> 
> Én tulajdonnévként értelmezném szívem szerint, bár nem széles körben használt? Ha rábólintasz így, akkor kiszedem, ha nem, akkor marad. 
> 
> Anna Nemeth-Szabo (anemeths)1:15 PM
> 
> OK, köszi, akkor nem kell változtatni / kijöhet.